packages/examples/packages/manage-state/src/index.test.ts

```diff
@@ -20,13 +20,13 @@ describe('onRpcRequest', () => {
     });
   });
 
-  describe('setState', () => {
+  describe('legacy_setState', () => {
```

Comment:
Can we test the new RPC methods too?

Reply:
I will add support to `snaps-jest` in a separate PR (but can target this PR), so it's easier to review.